napari/_qt/menus/file_menu.py

```diff
@@ -85,15 +85,15 @@ def __init__(self, window: 'Window'):
             {
                 'text': trans._('Copy Screenshot to Clipboard'),
                 'slot': window._qt_viewer.clipboard,
-                'shortcut': 'Alt+Shift+S',
+                'shortcut': 'Alt+C',
```

Comment:
because `Alt+Shift+S` is the shortcut for `Save Screenshot with Viewer...` (line 80 of this file) 

Every action needs to have a different shortcut.